src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/EvInputFactoryTest.groovy

```diff
@@ -41,7 +41,7 @@ class EvInputFactoryTest extends Specification implements FactoryTestHelper {
 
 		when:
 		Optional<EvInput> input = inputFactory.getEntity(
-				new SystemParticipantTypedEntityData<EvTypeInput>(parameter, inputClass, operatorInput, nodeInput, typeInput))
+				new SystemParticipantTypedEntityData<EvTypeInput>(parameter, inputClass,operatorInput, nodeInput, typeInput))
```

Comment:
:(

Reply:
@sensarmad don't get this - would you mind clarifying pls?!